build.gradle

```diff
@@ -361,16 +361,6 @@ android {
       applicationId = 'org.medicmobile.webapp.mobile.itech_malawi'
     }
 
-    cares_malawi_kch {
```

Comment:
Same here. Was `cares_malawi_kch`, `cares_malawi_mpc`, and `moh_togo_uat` supposed to be removed!